[TX Send] Fix `Send all funds` issue

bgptr opened the pull request:
Closes #2799

jholdstock commented:
Still seeing an issue here, to recreate:

- Paste a valid address into To field
- Type "1" into the Amount field. Hit backspace to delete it.
- Press "Send all" funds

Send button does not activate

bgptr commented:
Good catch, thank you. I've fixed it. 

Also, I found and fixed an issue too: 

- Send funds to another account
- Type valid amount
- Press "Send all" funds

After changing the `From` account the Tx details did not update. e.g the `Total amount sending` remained the same.
